docs/content/get-started/upload-launch.md

```diff
@@ -70,15 +70,17 @@ Uploaded assembly my_assembly to project unity_gdk successfully.
 
 > **TIP:** Depending on your network speed, the upload may take a little while (1-10 minutes) to complete.
 
+<br/>
 
 #### 3. Launch a cloud deployment
 
 The next step is to [start a cloud deployment](https://docs.improbable.io/reference/latest/shared/deploy/deploy-cloud#5-deploy-the-project) using the assembly that you just uploaded. When you launch a cloud deployment you have to provide four parameters:
 
-* **Assembly Name** - this identifies the worker assemblies to use. 
+* **Assembly Name** - this identifies the worker assemblies to use.
 * **Deployment Name** - this labels your deployment in the SpatialOS Console.
 * **Snapshot File** - this specifies which snapshot file the deployment uses. A snapshot declares the state of the world as the deployment starts.
 * **Launch Config File** - this is the deployment's launch configuration `.json` file which declares the world and load balancing configuration.
+* **Deployment Region** - the region that your deployment will be started in, set to `US` by default.
```

Comment:
```suggestion
* **Deployment Region** - the geographic location that your deployment starts in: `EU` for Europe or `US` for USA. Set to `US` by default.
```
Make it clear this is geography and the real-world as we talk about worlds a lot and region could mean region of the SpatialOS world or your game world.
Also made it into active voice and present simple so the text is stronger and less convoluted.